### PR TITLE
fix deepflavour discriminator

### PR DESCRIPTION
**Pull request by mverzett**
One-line fix of the DeepFlavour discriminator.

### CODE REVIEW

File: PhysicsTools/NanoAOD/python/jets_cff.py [hunk 1]
```diff
@@ -191,7 +191,7 @@
         nElectrons = Var("?hasOverlaps('electrons')?overlaps('electrons').size():0", int, doc="number of electrons in the jet"),
         btagCMVA = Var("bDiscriminator('pfCombinedMVAV2BJetTags')",float,doc="CMVA V2 btag discriminator",precision=10),
         btagDeepB = Var("bDiscriminator('pfDeepCSVJetTags:probb')+bDiscriminator('pfDeepCSVJetTags:probbb')",float,doc="DeepCSV b+bb tag discriminator",precision=10),
-        btagDeepFlavB = Var("bDiscriminator('pfDeepFlavourJetTags:probb')+bDiscriminator('pfDeepFlavourJetTags:probbb')",float,doc="DeepFlavour b+bb tag discriminator",precision=10),
+        btagDeepFlavB = Var("bDiscriminator('pfDeepFlavourJetTags:probb')+bDiscriminator('pfDeepFlavourJetTags:probbb')+bDiscriminator('pfDeepFlavourJetTags:problepb')",float,doc="DeepFlavour b+bb tag discriminator",precision=10),
```
Comment (gpetruc) [inline PhysicsTools/NanoAOD/python/jets_cff.py]: better change also the doc in b+bb+lepb ?

**Comment by arizzi** [comment]
from which version of MiniAOD we should apply this definition ?

File: PhysicsTools/NanoAOD/python/jets_cff.py (hunk 1 above)
Comment (mverzett) [reply PhysicsTools/NanoAOD/python/jets_cff.py]: OK

**Comment by mverzett** [comment]
Any version which has DeepFlavour

**Comment by arizzi** [comment]
ah, ok the fix is only for deepFlav, not deepCSV ..

**Pull request merged**